server/handlers.go

```diff
@@ -387,17 +391,52 @@ func (s *Server) handleConnectorCallback(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	w.Header().Set("Location", redirectURL)
```

Comment:
why setting header here, I believe `Redirect` will do this for u?

Reply:
oh yes, It will be deleted